src/bot/utils.ts

```diff
@@ -13,6 +14,10 @@ import {
 import { diffWords } from 'diff';
 import { APIEmbed } from 'discord-api-types/v10';
 import { godfatherRoleId } from './constants';
+import prisma from '../prisma';
+import { Category, UnsignedJoke } from 'typings';
+import { ProposalType } from '@prisma/client';
+import { jokeById, jokeByQuestion } from 'controllers';
```

Comment:
Attention a l'autocompletion :)  
```suggestion
import { jokeById, jokeByQuestion } from '../controllers';
```

Reply:
Bien vu ! Merci !